### ISSUE TITLE
ToRootTarget is not mapped in IEnumerable's

### ISSUE THREAD

**Issue by SlowLogicBoy** [agileobjects/AgileMapper#69]
When I map `IEnumerable<FromModel>` to `IEnumerable<ToModel>` with mapper configured with `ToRootTarget()`, `ToModel` instances have `SubPropertyA` and `SubPropertyB` == null
Code to reproduce: (used [dotnet-script](https://github.com/filipw/dotnet-script))
```csharp
#r "nuget: AgileObjects.AgileMapper, *"
#r "nuget: Bogus, *"
using AgileObjects.AgileMapper;

var lorem = new Bogus.DataSets.Lorem(); //For model generation

//Configure Mapper
var mapper = Mapper.CreateNew();
mapper.WhenMapping
    .From<FromModel>()
    .To<ToModel>()
    .Map((s, t) => s.FromSubModel)
    .ToRootTarget();

//Generate FromModels
var fromModels = Enumerable.Range(0, 100)
    .Select(i => new FromModel {
        PropertyA = lorem.Word(),
        PropertyB = lorem.Word(),
        FromSubModel = new FromSubModel{
            SubPropertyA = lorem.Word(),
            SubPropertyB = lorem.Word()
        }
    });

//Map
var toModels = mapper.Map(fromModels).ToANew<IEnumerable<ToModel>>();
Debugger.Break();

class FromModel
{
    public string PropertyA { get; set; }
    public string PropertyB { get; set; }
    public FromSubModel FromSubModel { get; set; }
}

class FromSubModel
{
    public string SubPropertyA { get; set; }
    public string SubPropertyB { get; set; }
}

class ToModel
{
    public string PropertyA { get; set; }
    public string PropertyB { get; set; }
    public string SubPropertyA { get; set; }
    public string SubPropertyB { get; set; }
}
```

**Comment by SteveWilkes** [comment]
Fixed in latest code, release to follow.

I have renamed the method to `ToTarget`, implying that the configured source will be applied to any level of a mapping.

**Comment by SteveWilkes** [comment]
Fixed in [v0.25](https://www.nuget.org/packages/AgileObjects.AgileMapper). Thanks again for the feedback!

**Comment by SlowLogicBoy** [comment]
Works perfectly. Thanks for the fast response.
